frontend/javascripts/oxalis/view/right-menu/mapping_info_view.js

```diff
@@ -126,24 +127,57 @@ class MappingInfoView extends React.Component<Props, State> {
   }
 
   renderIdTable() {
+    const {
+      mapping,
+      isMappingEnabled,
+      mappingColors,
+      activeViewport,
+      mousePosition,
+      zoomStep,
+      position,
+      dataset,
+      segmentationLayer,
+      renderMissingDataBlack,
+    } = this.props;
     const cube = this.getSegmentationCube();
-    const hasMapping = this.props.mapping != null;
-    const customColors = this.props.isMappingEnabled ? this.props.mappingColors : null;
+    const hasMapping = mapping != null;
+    const customColors = isMappingEnabled ? mappingColors : null;
 
     let globalMousePosition;
-    if (this.props.mousePosition && this.props.activeViewport !== OrthoViews.TDView) {
-      const [x, y] = this.props.mousePosition;
+    if (mousePosition && activeViewport !== OrthoViews.TDView) {
+      const [x, y] = mousePosition;
       globalMousePosition = calculateGlobalPos({ x, y });
     }
-
-    const getIdForPos = pos => pos && cube.getDataValue(pos, null, this.props.zoomStep);
+    let usableZoomStep = zoomStep;
+    const flycamPosition = position;
+    const resolutions = getResolutions(dataset);
+    // While render missing data black is not active and there is no segmentation for the current zoom step,
+    // the segmentation of a higher zoom step is shown.
+    // Here we determine the the next zoom step of the displayed segmentation data to get the correct segment ids.
+    while (
+      !renderMissingDataBlack &&
+      usableZoomStep < resolutions.length - 1 &&
+      !cube.hasDataAtPositionAndZoomStep(flycamPosition, usableZoomStep)
```

Comment:
The same has to be done for the mouse position, since that could differ in theory. This means, the "Segmentation resolution" has to be rendered in each cell (or as an additional column maybe). Also, I'd use "Magnification" as a label in the UI, since this fits better, I think. Our usages of `resolution` in the code are a bit unfortunate, I think, but since it's spread all over the place, I wouldn't change it now.